binr/rabin2/Makefile

```diff
@@ -3,6 +3,7 @@ BINDEPS=r_magic r_core r_db r_bin r_egg r_cons
 BINDEPS+=r_config r_io r_flags r_asm
 BINDEPS+=r_debug r_hash r_lang r_anal r_parse r_bp r_reg
 BINDEPS+=r_search r_syscall r_diff r_socket r_fs r_magic r_util
+CLFAGS+=-ldl
```

Comment:
This is linux specific. There's an acr variable for that. This will break static builds, crosscompilation and other platform too